src/mozanalysis/metrics/__init__.py

```diff
@@ -71,15 +73,15 @@ class DataSource:
     def _check_experiments_column_type(self, attribute, value):
         if value not in self.EXPERIMENT_COLUMN_TYPES:
             raise ValueError(
-                f"experiments_column_type {repr(value)} must be one of: "
-                f"{repr(self.EXPERIMENT_COLUMN_TYPES)}"
+                f"experiments_column_type {value!r} must be one of: "
+                f"{self.EXPERIMENT_COLUMN_TYPES!r}"
```

Comment:
This change seems unnecessary to me (and possibly less obvious what it's doing), but it is a little more succinct if you're familiar with `!r`/`!s`/`!a`. What do you think? It's RUF010 just for reference.

Reply:
I like it but I'm not married to it.  the `!r` format is snappy but it is a bit hard to figure out what is going on if you haven't seen it before

Reply:
I guess if ruff is doing it automatically for us it doesn't hurt. I think I'd lean slightly in favor of keeping it.